neurom/apps/tests/test_morph_stats.py

```diff
@@ -51,7 +51,7 @@
 }
 
 REF_OUT = {
-    'mean_soma_radius': 0.17071067811865476,
+    'mean_soma_radius': 0.13065629648763766,
```

Comment:
Because the soma type changed for the SWC files, all the area/volume comparisons have to be updated